modules/examiner/test/TestPlan.md

```diff
@@ -8,4 +8,15 @@
 4. Check "Clear Filters" button works well.
    [Automation Testing]
 5. Test Add a new examiner.
-   [Automation Testing]
+   [Automation Testing]  
+    a. Check that if you try to add an examiner with no name or no site, a warning message appears.  
```

Comment:
Is Project a filter option yet in this module?  If so, add tests for these permissions.

Reply:
No, Project is not a filter option.